### PR TITLE
Use z for pointer literals

### PR DESCRIPTION
**Pull request by njhale**
Replace pointer literal tricks and local helpers with [z.P](https://github.com/acorn-io/z/blob/78afed0850a8/z.go#L27).

### CODE REVIEW

**Review by tylerslaton — APPROVED** [review]

**Comment by njhale** [comment]
fixed a goimports issue, requesting re-review

**Review by iwilltry42 — APPROVED** [review]
:heart:

**Review by tylerslaton — APPROVED** [review]

File: pkg/config/config_test.go [hunk 1]
```diff
@@ -41,15 +42,15 @@ func TestAcornDNSStates(t *testing.T) {
 			// acornDNS is explicitly disabled, expect no clusterDomain to be returned
 			name: "acornDNS disabled expect no clusterdomains",
 			conf: &apiv1.Config{
-				AcornDNS: &[]string{"disabled"}[0], // hack for one-liner string pointer
+				AcornDNS: z.P("disabled"), // hack for one-liner string pointer
```
Comment (g-linville) [inline pkg/config/config_test.go:45]: nit: you can remove the comment here

```suggestion
				AcornDNS: z.P("disabled"),
```

**Review by g-linville — APPROVED** [review]

File: pkg/config/config_test.go (hunk 1 above)
Comment (njhale) [reply pkg/config/config_test.go]: done

**Pull request merged**